allensdk/test/brain_observatory/behavior/test_trials_processing.py

```diff
@@ -78,5 +78,159 @@ def test_calculate_reward_rate(kwargs, expected):
     assert np.allclose(
         trials_processing.calculate_reward_rate(**kwargs),
         expected, 
-    ), "calculared reward rate should match expected reward rate :("
+    ), "calculated reward rate should match expected reward rate :("
 
+def test_trial_data_from_log():
```

Comment:
as-is this test will error out on the first missed assertion. I might restructure it like:
```
def data_and_expectation_one():
    ...
    return data, expectation

def data_and_expectation_two():
    ...
    return data, expectation

@pytest.mark.parametrize("data_exp_getter", [
    data_and_expectation_one, 
    data_and_expectation_two
])
def test_trial_data_from_log(data_exp_getter):
    data, expectation = data_exp_getter()
    assert trials_processing.trial_data_from_log(data) == expectation
```
then pytest will generate seperate tests for each case that can pass and fail independently

Reply:
Done.